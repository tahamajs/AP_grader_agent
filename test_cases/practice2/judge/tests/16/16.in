enter 1
enter 2
show_table_info 4
reserve_table 1 1
exit 1
show_table_info 1
enter 1
