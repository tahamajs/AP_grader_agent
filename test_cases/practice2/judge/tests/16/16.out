Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Table ID: 4
People at the table:
Table remaining capacity: 2
Waiting queue length: 0
Paria sits at table 1
Paria exits!
Table ID: 1
People at the table:
Table remaining capacity: 4
Waiting queue length: 0
Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
